packages/carbon-graphs/src/js/helpers/constants.js

```diff
@@ -316,4 +316,5 @@ export default {
   DEFAULT_LEGEND_LINE_WIDTH: 28,
   DEFAULT_LEGEND_LINE_WIDTH_WITH_SYMBOL: 18,
   LEGEND_LINE_POSITION: 1.5,
+  DEFAULT_OVERLAPPING_PADDING: 5,
```

Comment:
This needs to be reviewed by UX:
cc: @eawww

Reply:
Assuming this is relative to the bottom of the tick labels, the typical value seems to be half of this. I'd like to see what 2.5 looks like.